.circleci/config.yml

```diff
@@ -378,6 +377,10 @@ jobs:
       xcode: "10.1.0"
     shell: /bin/bash --login
     working_directory: /Users/distiller/wp-desktop
+    environment:
+      BRANCHNAME: << pipeline.git.branch >>
```

Comment:
Same as prior comment - if we stick to `CIRCLE_BRANCH`, do we need a separate `BRANCHNAME` var? Seems like we could remove this?